src/frontend/planner_test/tests/testdata/input/range_scan.yaml

```diff
@@ -428,3 +428,54 @@
   expected_outputs:
   - logical_plan
   - batch_plan
+- name: When OR clauses contain non-overlapping conditions,, we can pushdown serveral scan_range.
+  before:
+  - create_table_and_mv
```

Comment:
Suggests adding some tests in descending order.

Reply:
Good catch, We don't need to consider the direction of the ordering in the range comparison in this pr, so we change all the defaults to `asce` to simplify the comparison logic.